docker-devman

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e
+DOT="$(dirname $(readlink -f ${BASH_SOURCE[0]}))"
+
+if [ -f "$(pwd)/devman.conf" ];
+then
+  conf="-v $(pwd)/devman.conf:/app/devman.conf:ro"
+elif [ -f "$DOT/devman.conf" ]
+then
+  conf="-v $DOT/devman.conf:/app/devman.conf:ro"
+else
+  conf=""
+fi
+
+if [ -f "$(pwd)/repos.yaml" ];
+then
+  repos="-v $(pwd)/repos.yaml:/app/repos.yaml:ro"
+elif [ -f "${DOT}/repos.yaml" ];
+then
+  repos="-v ${DOT}/repos.yaml:/app/repos.yaml:ro"
+else
+  repos=""
+fi
+
+if [ -f $HOME/kronos ]
+then
+  echo "~/kronos exists as file, will not destroy it"
```

Comment:
Ça veut rien dire pour un user ça. T'es senser faire quoi quand tu vois ce message là ? Je te suggère d'expliquer la situation et d'indiquer ce qu'un user doit effectuer pour aller plus loin.

Reply:
please move it using `mv ~/kronos ~/kronos.old`